views/index.hbs

```diff
@@ -1,10 +1,500 @@
-<html>
-  <head>
-    <title>Jenkins Plugins List</title>
-    {{> header }}
-  </head>
-  <body>
-    <div id="grid-box"></div>
-    <script type="text/javascript" src="/main.js"></script>
-  </body>
+<!DOCTYPE html>
```

Comment:
this file gets overridden every 15 minutes, so hopefully you didn't actually make any changes did you?

Reply:
No. It's the automatically downloaded file.